server/eclipse-project/src/main/java/au/org/democracydevelopers/corla/endpoint/GenerateAssertions.java

```diff
@@ -21,5 +21,23 @@
 
 package au.org.democracydevelopers.corla.endpoint;
 
-public class GenerateAssertions {
+import spark.Request;
+import spark.Response;
+import us.freeandfair.corla.endpoint.AbstractDoSDashboardEndpoint;
+
+public class GenerateAssertions extends AbstractDoSDashboardEndpoint {
+    @Override
+    protected String endpointBody(Request the_request, Response the_response) {
+        return "";
```

Comment:
I'm guessing this will be fleshed out after this PR gets merged?

Reply:
Right - the Generate Assertions endpoint is my next card.